service/internal/changedetection/components_test.go

```diff
@@ -0,0 +1,455 @@
+							Version:               "1.6.0",
+						},
+						{
+							Catalog:               "",
+							Name:                  "aws-operator",
+							Reference:             "",
+							ReleaseOperatorDeploy: false,
+							Version:               "8.7.6",
+						},
+						{
+							Catalog:               "",
+							Name:                  "calico",
+							Reference:             "",
+							ReleaseOperatorDeploy: false,
+							Version:               "3.10.4",
+						},
+						{
+							Catalog:               "",
+							Name:                  "cert-operator",
+							Reference:             "",
+							ReleaseOperatorDeploy: false,
+							Version:               "0.1.0",
+						},
+						{
+							Catalog:               "",
+							Name:                  "cluster-operator",
+							Reference:             "",
+							ReleaseOperatorDeploy: false,
+							Version:               "2.3.2",
+						},
+						{
+							Catalog:               "",
+							Name:                  "containerlinux",
+							Reference:             "",
+							ReleaseOperatorDeploy: false,
+							Version:               "2512.2.1",
+						},
+						{
+							Catalog:               "",
+							Name:                  "etcd",
+							Reference:             "",
+							ReleaseOperatorDeploy: false,
+							Version:               "3.4.9",
+						},
+						{
+							Catalog:               "",
+							Name:                  "kubernetes",
+							Reference:             "",
+							ReleaseOperatorDeploy: false,
+							Version:               "1.16.13",
+						},
+					},
+				},
+			},
+			targetRelease: releasev1alpha1.Release{
+				Spec: releasev1alpha1.ReleaseSpec{
+					Components: []releasev1alpha1.ReleaseSpecComponent{
+						{
+							Catalog:               "",
+							Name:                  "app-operator",
+							Reference:             "",
+							ReleaseOperatorDeploy: false,
+							Version:               "1.0.0",
+						},
+						{
+							Catalog:               "",
+							Name:                  "aws-cni",
+							Reference:             "",
+							ReleaseOperatorDeploy: false,
+							Version:               "1.6.0",
+						},
+						{
+							Catalog:               "",
+							Name:                  "aws-operator",
+							Reference:             "",
+							ReleaseOperatorDeploy: false,
+							Version:               "8.7.7",
+						},
+						{
+							Catalog:               "",
+							Name:                  "calico",
+							Reference:             "",
+							ReleaseOperatorDeploy: false,
+							Version:               "3.10.4",
+						},
+						{
+							Catalog:               "",
+							Name:                  "cert-operator",
+							Reference:             "",
+							ReleaseOperatorDeploy: false,
+							Version:               "0.1.0",
+						},
+						{
+							Catalog:               "",
+							Name:                  "cluster-operator",
+							Reference:             "",
+							ReleaseOperatorDeploy: false,
+							Version:               "2.3.3",
```

Comment:
Let's get a comment that this is the version that should trigger the change.

Reply:
Added comments, where we expect to see an update for components